### PR TITLE
Use assembly in dbSNP imports

### PR DESCRIPTION
**Pull request by helensch**

### CODE REVIEW

File: scripts/import/dbSNP_v2/load_dbsnp.pl [hunk 1]
```diff
@@ -92,9 +92,11 @@
 my ($num_lines) = parse_dbSNP_file($config);
 
 # Close open filehandles
-for my $ma_type ('update', 'log') {
-  my $fh = $config->{join('-', 'ma', $ma_type, 'fh')};
-  $fh->close();
+if ($config->{'assembly'} eq 'GRCh38') {
```
Comment (sarahhunt) [inline scripts/import/dbSNP_v2/load_dbsnp.pl]: A comment here would be helpful

File: scripts/import/dbSNP_v2/load_dbsnp.pl [hunk 2]
```diff
@@ -129,19 +131,25 @@ sub init_reports {
     die("$config->{'error_file'} already exists. Please rename or delete\n");
   }
 
+  # For GRCh38 the 1000Genomes minor allele needs flipping
```
Comment (sarahhunt) [inline scripts/import/dbSNP_v2/load_dbsnp.pl]: This suggests they all need reverse complimenting

File: scripts/import/dbSNP_v2/load_dbsnp.pl [hunk 3]
```diff
@@ -1587,6 +1616,16 @@ sub get_seq_region_names {
   return (\%seq_region);
 }
 
+# For a given chr look up the seq_region
+sub get_seq_region_chr {
+  my ($dbh, $chr) = @_;
```
Comment (sarahhunt) [inline scripts/import/dbSNP_v2/load_dbsnp.pl]: This relies on only one seq_region record for Y being imported from core to variation. We should probably check somewhere.

File: scripts/import/dbSNP_v2/load_dbsnp.pl (hunk 1 above)
Comment (helensch) [reply scripts/import/dbSNP_v2/load_dbsnp.pl]: A comment has been added.

File: scripts/import/dbSNP_v2/load_dbsnp.pl (hunk 3 above)
Comment (helensch) [reply scripts/import/dbSNP_v2/load_dbsnp.pl]: A comment has been added and a ticket created to check this at the start of the script.

File: scripts/import/dbSNP_v2/load_dbsnp.pl (hunk 2 above)
Comment (helensch) [reply scripts/import/dbSNP_v2/load_dbsnp.pl]: A comment has been added

**Pull request merged**